src/components/Countdown.js

```diff
@@ -1,12 +1,47 @@
-import React from "react";
+import React, { useEffect, useState } from "react";
+
+const calculateTimeLeft = () => {
+  let year = new Date().getFullYear();
+  const endTime = new Date(year, 1, 21, 9, 30, 0, 0); // Feb 21, 9:30am
+  const difference = +endTime - +new Date();
+  let timeLeft = {};
+
+  if (difference > 0) {
+    timeLeft = {
+      days: Math.floor(difference / (1000 * 60 * 60 * 24)),
+      hours: Math.floor((difference / (1000 * 60 * 60)) % 24),
+      minutes: Math.floor((difference / 1000 / 60) % 60),
+      seconds: Math.floor((difference / 1000) % 60),
+    };
+  }
+
+  return timeLeft;
+};
+
+const useCounter = () => {
+  const [timeLeft, setTimeLeft] = useState(calculateTimeLeft());
+
+  useEffect(() => {
+    setTimeout(() => {
+      setTimeLeft(calculateTimeLeft());
+    }, 1000);
+  }, [timeLeft]);
```

Comment:
This is super minor, so feel free to ignore! Instead of having this loop and recalculate because it sets timeLeft with useEffect, you can also just use setInterval (imo makes more sense). it would look like this:

```
useEffect(() => {
    const interval = setInterval(() => setTimeLeft(calculateTimeLeft()), 1000)
    return () => clearInterval(interval);
  }, []);
  ```
  
  this also clears the interval on unmount